src/test/java/uk/gov/hmcts/reform/migration/service/RespondentsDataMigrationServiceTest.java

```diff
@@ -0,0 +1,303 @@
+        Respondent expectedRespondent = createNewRespondent();
+
+        assertThat(respondents).hasSize(1);
+        assertThat(firstRespondentParty.get("partyType")).isNull();
+        assertThat(firstRespondentParty.get("firstName")).isNull();
+        assertThat(firstRespondentParty.get("lastName")).isNull();
+        assertThat(firstRespondentParty.get("dateOfBirth")).isNull();
+        assertThat(objectMapper.convertValue(firstRespondentParty.get("address"), Address.class)).isEqualTo(Address.builder()
+            .addressLine1(null)
+            .addressLine2(null)
+            .addressLine3(null)
+            .postcode("postcode")
+            .county("Kent")
+            .country(null)
+            .postTown(null)
+            .build());
+        assertThat(firstRespondentParty.get("telephoneNumber")).isEqualTo(objectMapper.convertValue(TelephoneNumber.builder().build(), Object.class));
+        assertThat(firstRespondentParty.get("email")).isNull();
+        assertThat(firstRespondentParty.get("gender")).isNull();
+        assertThat(firstRespondentParty.get("genderIdentification")).isNull();
+        assertThat(firstRespondentParty.get("placeOfBirth")).isNull();
+        assertThat(firstRespondentParty.get("relationshipToChild")).isNull();
+        assertThat(firstRespondentParty.get("contactDetailsHidden")).isNull();
+        assertThat(firstRespondentParty.get("litigationIssues")).isNull();
+        assertThat(firstRespondentParty.get("litigationIssuesDetails")).isNull();
+    }
+
+    @Test
+    public void whenOldRespondentWithOneNameIsMigratedShouldReturnNewListStructureWithOnlyFirstName() {
+        Map<String, Object> data = new HashMap<>();
+
+        OldRespondent respondent = OldRespondent.builder()
+            .name("Beyoncé")
+            .build();
+
+        data.put("respondents", OldRespondents.builder()
+            .firstRespondent(respondent)
+            .build());
+
+        CaseDetails caseDetails = CaseDetails.builder()
+            .id(1111L)
+            .data(data)
+            .build();
+
+        service.migrate(caseDetails);
+
+        List<Map<String, Object>> respondents = (List<Map<String, Object>>) caseDetails.getData().get("respondents1");
+
+        Map<String, Object> firstRespondent = (Map<String, Object>) respondents.get(0).get("value");
+        Map<String, Object> firstRespondentParty = (Map<String, Object>) firstRespondent.get("party");
+
+        assertThat(respondents).hasSize(1);
+        assertThat(firstRespondentParty.get("firstName")).isEqualTo("Beyoncé");
+        assertThat(firstRespondentParty.get("lastName")).isNull();
+    }
+
+    @Test
+    public void whenOldRespondentWithManyNamesIsMigratedShouldReturnNewListStructureWithMultipleFirstNames() {
+        Map<String, Object> data = new HashMap<>();
+
+        OldRespondent respondent = OldRespondent.builder()
+            .name("Jean Paul Gautier")
```

Comment:
Logic could have been extracted to external class that would:
1. open possibility for reuse
2. make testing easier

Not important for that branch however an idea for for the future.

Reply:
happy to raise this in an separate PR that can go onto master